### ISSUE TITLE
Support Conjure generators on local project

### ISSUE THREAD

**Issue by tdeitch** [palantir/gradle-conjure#90]
Right now if I have a project called `my-service-api`, I can create a new project called `my-service-local-api`, add `conjure-local`, pull down my published Conjure IR from `my-service-api`, and use that to generate, say, a Postman collection.

Instead, I'd like to add some plug-in(s) to my existing `my-service-api` project that give(s) me a `gradle generateConjurePostman` task that I can run right from my project to generate a Postman collection from the current local Conjure files.

cc @ferozco

**Comment by martijnarts** [comment]
I've solved this hackily with this `my-service-postman/build.gradle`:

```
apply plugin: 'com.palantir.conjure-local'

task copyConjureIr(type: Copy) {
    from '../my-service-api/build/conjure-ir'
    into './postman/build/tmp/'
    include 'my-service-api.conjure.json'
    rename 'my-service-api.conjure.json', 'my-service-api-0.0.0.conjure.json'
}

dependencies {
    conjure files('./postman/build/tmp/my-service-api-0.0.0.conjure.json')
    conjureGenerators 'com.palantir.conjure.postman:conjure-postman:0.1.0'
}

tasks.generatePostman.dependsOn(copyConjureIr)
```

**Comment by ferozco** [comment]
We should be able to solve this by adding support for generic generators to `com.palantir.conjure`

**Comment by ferozco** [comment]
Closed by #148
